openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -1,404 +1,291 @@
+        return super(CreateRenderlayer, self).imprint_instance_node(node,
+                                                                    data=data)
 
-        Warning:
-            Disabling SSL certificate validation is defeating one line
-            of defense SSL is providing and it is not recommended.
+    def remove_instances(self, instances):
+        """Remove specified instances from the scene.
+
+        This is only removing `id` parameter so instance is no longer
+        instance, because it might contain valuable data for artist.
 
         """
-        if "verify" not in kwargs:
-            kwargs["verify"] = not os.getenv("OPENPYPE_DONT_VERIFY_SSL", True)
-        return requests.get(*args, **kwargs)
+        # Instead of removing the single instance or renderlayers we instead
+        # remove the CreateRender node this creator relies on to decide whether
+        # it should collect anything at all.
+        nodes = self._get_singleton_node(return_all=True)
+        if nodes:
+            cmds.delete(nodes)
+
+        # Remove ALL the instances even if only one gets deleted
+        for instance in list(self.create_context.instances):
+            if instance.get("creator_identifier") == self.identifier:
+                self._remove_instance_from_context(instance)
+
+                # Remove the stored settings per renderlayer too
+                node = instance.data.get("instance_node")
+                if node and cmds.objExists(node):
+                    cmds.delete(node)
+
+    def get_instance_attr_defs(self):
+        """Create instance settings."""
+
+        return [
+            BoolDef("review",
+                    label="Review",
+                    tooltip="Mark as reviewable",
+                    default=True),
+            BoolDef("extendFrames",
+                    label="Extend Frames",
+                    tooltip="Extends the frames on top of the previous "
+                            "publish.\nIf the previous was 1001-1050 and you "
+                            "would now submit 1020-1070 only the new frames "
+                            "1051-1070 would be rendered and published "
+                            "together with the previously rendered frames.\n"
+                            "If 'overrideExistingFrame' is enabled it *will* "
+                            "render any existing frames.",
+                    default=False),
+            BoolDef("overrideExistingFrame",
+                    label="Override Existing Frame",
+                    tooltip="Override existing rendered frames "
+                            "(if they exist).",
+                    default=True),
+
+            # TODO: Should these move to submit_maya_deadline plugin?
+            # Tile rendering
+            BoolDef("tileRendering",
+                    label="Enable tiled rendering",
+                    default=False),
+            NumberDef("tilesX",
```

Comment:
As far as I know you can't. Right @iLLiCiTiT - I'd definitely love to be able to do that though.

Reply:
Yeah it would be awesome to steal Houdini's syntax for setting up disable/visible conditionals https://www.sidefx.com/docs/houdini19.0/ref/windows/optype.html#conditions